src/main/resources/mcmod.info

```diff
@@ -2,18 +2,11 @@
 {
   "modid": "EnderStorage",
   "name": "EnderStorage",
-  "description": "Stores your stuff in the END!
-
-Credits: Ecu - original idea, design,
-        chest and pouch texture
-    Rosethorns - tank model
-    Soaryn - tank texture
-
-Supporters: Blkdragon112",
+  "description": "Stores your stuff in the END!\nCredits: Ecu - original idea, design,\nchest and pouch texture\nRosethorns - tank model\nSoaryn - tank texture\n\nSupporters: Blkdragon112",
   "version": "${version}",
   "mcversion": "${mc_version}",
   "url": "http://www.minecraftforum.net/topic/909223",
   "authorList": [ "ChickenBones, covers1624" ],
   "dependencies": [ "CodeChickenCore" ]
 }
-]
\ No newline at end of file
+]
```

Comment:
I'm not sure how the file got without a newline at the end as Git usually automatically adds these. I added it to prevent any unexpected behavior as per [this](https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file) thread.